test/DbPostgreSQLAdapterTest/Tests/DbConnectionTest.cpp

```diff
@@ -0,0 +1,41 @@
+#include "stdafx.h"
+
+#include "Connection.h"
+#include "ConnectionConfiguration.h"
+#include "DbAdapterInterface/IDatabase.h"
+#include "Helpers/Helpers.h"
+#include "Helpers/DefaultConnectionConfiguration.h"
+
+
+using namespace testing;
+namespace systelab::db::postgresql::unit_test {
+
+	class DbConnectionTest : public Test
+	{
+		void SetUp() override
+		{
+			dropDatabase(defaultDbName);
```

Comment:
The drop on SetUp is a just in case situation and enough in loop, but I would do it also on **TearDown** to leave stuff clean at the end of each test.

Reply:
Good idea, I'll change it